### DISCUSSION TITLE
All Bundled

### DISCUSSION THREAD

**Discussion by JackW6809** [ovh/manager#5306] [Q&A] [answered]
Hello,

I was wondering if there is an application that I can run for the whole panel, not just certain sections?

Thanks,
Jack W.

**Comment by antleblanc** [comment, marked as answer]
Hi @JackW6809!

Unfortunately there's no such application. The Control Panel is a set a multiple
applications tightly coupled to a "Universe" (e.g: Web/Dedicated/Public Cloud or Telecom).
